Lina Update (#51)
IllusionMods/KoikatsuStoryTranslation: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-14/translation.txt

> ﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about this Sensei's experience with men? My, are you alright with just my experience with men? You don't know if I might have some experience with women too, right?

Reviewer: This one seems a bit awkward, still.  I tried reworking. Either is fine.
```suggestion
﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about Sensei's experience with men? Oh, are you sure I only have experience with men? Maybe I've been with women, too, right?

```

> ﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about this Sensei's experience with men? My, are you alright with just my experience with men? You don't know if I might have some experience with women too, right?

Reviewer: This one seems a bit awkward, still.  I tried reworking. Either is fine.
```suggestion
﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about Sensei's experience with men? Oh, are you sure I've only had men? Maybe I've been with women, too, right?

```
